File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex19.c
#include <stdio.h>

main()
{
    printf("Peso ideal\n\n");

    //vars
    float altura, peso;
    char sexo[1];

    //Entrada
    printf("Insira sua altura em metros: ");
    scanf("%f",&altura);

    printf("Insira o seu sexo (F/M): ");
    scanf("%s",sexo);

    //Processamento
    if ( sexo == 'M' )
        peso = ( 72.7 * altura ) - 58;
    else
        peso = ( 62.1 * altura ) - 44.7;

    printf("\nPeso ideal: %.1fKg",peso);



}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex19.java
//Importando Scanner
import java.util.Scanner;

public class Ex19 {
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Peso ideal para uma pessoa\n\n");
		
		//vars
		int sexo;
		double peso, altura;
		
		//Entrada
		System.out.print("Insira sua altura em metros: ");
		altura = ent.nextDouble();
		
		System.out.println("Sexo");
		System.out.println("1 - Feminino");
		System.out.println("2 - Masculino");
		System.out.print("Insira o número correspondente ao seu sexo: ");
		sexo = ent.nextInt();
		
		//Processamento
		if ( sexo == 1 )
			peso = 62.1 * altura - 44.7;
		else
			peso = 72.7 * altura - 58;
		
		//Saída
		System.out.printf("Peso ideal: %.1f", peso);
	}
}
